stdlib/3/urllib/request.pyi

```diff
@@ -58,8 +58,10 @@ class Request:
     selector: str
     data: Optional[bytes]
     headers: Dict[str, str]
+    unredirected_hdrs: Dict[str, str]
     unverifiable: bool
     method: Optional[str]
+    timeout: Optional[float]  # Undocumented
```

Comment:
Could you also add a comment that this field is set externally and not always available?

Reply:
Done with 85238ec11f6ffdc3d5673e5869b874b243bb21af